server/lib/src/utils/element_helper.dart

```diff
@@ -492,8 +486,10 @@ class ElementHelper {
     Duration? settleBetweenScrollsTimeout,
     Duration? dragDuration,
   }) async {
-    delta ??= defaultScrollDelta;
-    maxScrolls ??= defaultScrollMaxIteration;
+    delta ??= FlutterDriver.instance.settings
+        .getSetting('flutterScrollDelta');;
+    maxScrolls ??= FlutterDriver.instance.settings
+        .getSetting('flutterScrollMaxIteration');
```

Comment:
same as above

Reply:
Thanks or prompt review @sudharsan-selvaraj Incorporated the review comments